scripts/build_utils.sh

```diff
@@ -632,25 +632,18 @@ setup_pbuilder() {
 }
 
 use_ppa() {
+    # only use ppa on focal for reef+
     case $vers in
-        10.*)
-            # jewel
+        15.*) # o
             use_ppa=false;;
-        11.*)
-            # kraken
+        16.*) # p
             use_ppa=false;;
-        12.*)
-            # luminous
+        17.*) # q
```

Comment:
```suggestion
        17.2*) # q
```

as reef is not tagged yet. the commits in main are all named something like `v17.2.0-267-g8bae27e49f9`,

```console
$ git describe --tag
v17.2.0-267-g8bae27e49f9
```

and now that quincy is released, we are not likely to build for 17.1.x anymore. so `17.2*` is a safe choice.